### PR TITLE
upgrade to angular 17

### PR DESCRIPTION
**Pull request by allenkepler-finastra**
upgrade to angular 17

### CODE REVIEW

File: SampleApp/package.json [hunk 1]
```diff
@@ -1,51 +1,51 @@
...
+    "ng": "ng",
+    "start": "ng serve",
+    "build": "ng build",
+    "test": "ng test",
+    "lint": "ng lint",
+    "e2e": "ng e2e"
+  },
+  "private": true,
+  "dependencies": {
+    "@angular/animations": "^16.0.4",
```
Comment (picheli20) [inline SampleApp/package.json:14]: Might need to bump those version as well, it still as 16

File: SampleApp/package.json [hunk 2]
```diff
@@ -1,51 +1,51 @@
...
+    "@angular/compiler": "^16.0.4",
+    "@angular/core": "^16.0.4",
+    "@angular/forms": "^16.0.4",
+    "@angular/material": "^16.0.3",
+    "@angular/platform-browser": "^16.0.4",
+    "@angular/platform-browser-dynamic": "^16.0.4",
+    "@angular/router": "^16.0.4",
+    "core-js": "^3.21.0",
+    "hammerjs": "^2.0.8",
+    "ng2-pdfjs-viewer": "file:ng2-pdfjs-viewer-16.0.4.tgz",
```
Comment (picheli20) [inline SampleApp/package.json:26]: Also this guy, need to change to `ng2-pdfjs-viewer-17.0.2.tgz`

Comment (allenkepler-finastra) [reply SampleApp/package.json]: done

File: SampleApp/package.json (hunk 1 above)
Comment (allenkepler-finastra) [reply SampleApp/package.json]: done

**Pull request merged**